package.json

```diff
@@ -85,23 +84,23 @@
     "@types/node": "^13.13.1",
     "@types/node-forge": "^0.9.4",
     "@types/webpack": "^4.41.21",
+    "arkit": "^1.6.4",
     "copy-webpack-plugin": "^6.0.2",
     "documentation": "^8.1.2",
     "eslint": "^7.0.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-prettier": "^3.1.3",
     "file-loader": "^6.0.0",
-    "grpc-tools": "^1.9.0",
-    "grpc_tools_node_protoc_ts": "^4.1.0",
     "jest": "^25.0.0",
     "npm-dts": "^1.3.0",
     "prettier": "^2.0.5",
-    "protoc": "^1.0.4",
+    "serve": "^11.3.2",
```

Comment:
What's this for?

Reply:
This was for testing the relay functionality in QEMU before I integrated it into polykey, I've removed it now :+1: